voltdbclient/url_test.go

```diff
@@ -23,28 +23,33 @@ func TestParseURL(t *testing.T) {
 	}
 
 	for _, s := range sample {
-		u, err := parseURL(s.conn)
-		if s.pass {
-			if err != nil {
-				t.Fatal(err)
+		s := s //capture variable
+		t.Run("", func(t *testing.T) {
```

Comment:
actually parallell testing makes test faster. If we increase test cases, it could be helpful (might not be necessary with low test case but will be good in anycase)